app/code/Magento/ConfigurableProduct/Model/Product/Type/Collection/SalableProcessor.php

```diff
@@ -38,19 +38,20 @@ public function __construct(StatusFactory $stockStatusFactory)
      * when filter already added and therefore may break the collection.
      *
      * @param Collection $collection
+     * @param bool $isFilterInStock
      * @return Collection
      */
-    public function process(Collection $collection)
+    public function process(Collection $collection, $isFilterInStock = true)
```

Comment:
Please, adapt \Magento\ConfigurableProduct\Test\Unit\Model\Product\Type\Collection\SalableProcessorTest to cover new parameter cases